content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -183,7 +190,9 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
-3. Add the appropriate Kubernetes `apt` repository:
+3. Add the appropriate Kubernetes `apt` repository. Please note that this repository have packages
+   only for Kubernetes {{< skew currentVersion >}}; for other Kubernetes minor versions, you need to
+   add another repository that has the URL with your desired Kubernetes minor version.
```

Comment:
I wonder if this statement will actually help a new user to get the repository that they want.

Reply:
Fixed in the latest commit